bridge/api/api.go

```diff
@@ -82,6 +86,40 @@ func (b *Api) handleHealthcheck(c echo.Context) error {
 	return c.String(http.StatusOK, "OK")
 }
 
+func (b *Api) handleConfigReload(c echo.Context) error {
+	cfgURL := b.GetString("ConfigURL")
+	if cfgURL == "" {
+		b.Log.Warning("Reload API triggered, but no config file url set.")
+		return c.String(http.StatusInternalServerError, "Internal Server Error")
+	}
+
+	b.Log.Debugf("Reloading config from remote file: " + cfgURL)
+	_, err := url.ParseRequestURI(cfgURL)
+	if err != nil {
+		b.Log.Error("Malformed config file url: ", err)
+		return c.String(http.StatusInternalServerError, "Internal Server Error")
+	}
+	res, err := http.Get(cfgURL)
+	if err != nil {
+		b.Log.Error("Failed to fetch remote config file: ", err)
+		return c.String(http.StatusInternalServerError, "Internal Server Error")
+	}
+	defer res.Body.Close()
+	content, err := ioutil.ReadAll(res.Body)
+	if err != nil {
+		b.Log.Error("Error reading remote config file: ", err)
+		return c.String(http.StatusInternalServerError, "Internal Server Error")
+	}
+	cfgfile := b.GetConfigFile()
```

Comment:
Use https://godoc.org/github.com/spf13/viper#ConfigFileUsed

Reply:
Ugh. I don't know how I missed that.